_posts/en/posts/2023-12-06-release-26.0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Bitcoin Core 26.0 released
+name: blog-release-26.0
+id: en-blog-release-26.0
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 26.0 is now available.
+---
+Bitcoin Core version 26.0 is now available for [download][download
+page].  See the [release notes][] for more information about the many
```

Comment:
not sure how this markdown file works, but shouldn't this be:

```suggestion
page].  See the [release notes][release notes] for more information about the many
Comment

```

just based on how the other ones are.

Reply:
Looks like the link still works in either case (has been like this for the past release posts), but I've changed this for consistency.